Scenarios/Secure-Baseline/bicepWithAVM/03-Supporting-Services/main.bicep

```diff
@@ -5,6 +5,9 @@ targetScope = 'resourceGroup'
 /* -------------------------------------------------------------------------- */
 
 import { skuType as keyVaultSkuType } from './modules/key-vault/types.bicep'
+import { imageReferenceType, nicConfigurationType, osDiskType } from './modules/virtual-machine/types.bicep'
+import { containerRegistrySkuType } from './modules/container-registry/types.bicep'
```

Comment:
use skuType and an alias like for the key vault

Reply:
The `skuType` for Key Vault and for Container Registry differ in values. 

Key Vault takes the values `premium | standard | null`.
Container Registry takes the values `Premium | Standard | Basic | null`.

I will create a separate `skuType` inside `modules/container-registry/types.bicep` and import it using an alias.